app/javascript/controllers/slider_controller.js

```diff
@@ -0,0 +1,26 @@
+import { Controller } from "@hotwired/stimulus"
+import Splide from '@splidejs/splide';
+
+// Connects to data-controller="slider"
+export default class extends Controller {
+  static targets = [ "slider", "toggleFullscreen" ]
+
+  connect() {
+    const slider = this.sliderTarget
+    new Splide(slider, {
+      heightRatio: 2/3,
+      cover: true,
+      i18n: {
+        prev: 'Forrige',
+        next: 'Neste',
+        first: 'Første',
+        last: 'Siste',
+        slideX: 'Gå til %s',
+        pageX: 'Side %s'
```

Comment:
Googlet litt hvordan få til i18n i JS fra Rails. Det går visst an, men usikker på om det er verdt det atm. Hva tenker dere?

Reply:
Yep vi har dette i et annet prosjekt, er veldig fint å kunne ha all oversetting på ett sted. Så hvis du har lyst til å legge det til så go ahead :)

Reply:
Den løsningen jeg skisserer der er riktig måte å gjøre det på?

Reply:
I det andre prosjektet så har vi bare en variable som er `this.$t('')` det prosjektet bruker, lurer på om vi kan få til noe samme med stimulus?